docs/UserGuide.md

```diff
@@ -189,8 +198,8 @@ Example commands:
 * `delete n/Alex Yeoh`
 * `delete n/John Doe m/`
 
->:bulb: Specify the medical history to be deleted using `m/` if it's only the medical history data that is to be deleted
-e.g `delete n/John Doe m/Diabetes`
+>:bulb: Specify the medical history to be deleted using `m/` if it's the only medical history that needs to be deleted
+> e.g `delete n/John Doe m/Diabetes`
```

Comment:
Is this suppose to refer to the scenario where a user wants to delete only one of the two medical histories present?